### PR TITLE
update(bootstrap): 4.5 minor version

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- correct ScrollSpy.target definition
- minor version bump

see:
https://github.com/twbs/bootstrap/commit/22f75ca2e328ad232dd2afafd15dc0c79a908410
https://github.com/twbs/bootstrap/releases/tag/v4.5.0 (JavaScript para)

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## bootstrap ([<kbd>unpkg</kbd>](https://unpkg.com/browse/bootstrap@latest/))
was missing the following properties:
1. Alert
1. Button
1. Carousel
1. Collapse
1. Dropdown

<details>
<summary>as well as these 7 other properties...</summary>
<p>Modal, Popover, Scrollspy, Tab, Toast, Tooltip, Util</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 39dd259ee0feb88e41f6ba8d0da12fbf193218e7
</p>

File: types/bootstrap/index.d.ts [hunk 1]
```diff
@@ -223,7 +223,7 @@ export interface ScrollspyOption {
      *
      * @default ""
      */
-    target?: string | Element;
+    target?: string | JQuery | Element;
```
Comment (denisname) [inline types/bootstrap/index.d.ts:226]: `JQuery<Element>` may be better.

```suggestion
    target?: string | JQuery<Element> | Element;
```

File: types/bootstrap/bootstrap-tests.ts [hunk 2]
```diff
@@ -244,6 +244,10 @@ $("#scrollspy").scrollspy({
     target: document.getElementById("navbar-example2") as HTMLElement
 });
 
+$("#scrollspy").scrollspy({
+    target: $("navbar-example2"),
```
Comment (denisname) [inline types/bootstrap/bootstrap-tests.ts:248]:
```suggestion
    target: $("#navbar-example2"),
```

**Review by denisname — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge

**Pull request merged**